go/vt/vttablet/tabletmanager/vreplication/vreplicator.go

```diff
@@ -1030,3 +1032,37 @@ func (vr *vreplicator) newClientConnection(ctx context.Context) (*vdbClient, err
 	}
 	return dbClient, nil
 }
+
+// setExistingRowsCopied deals with the case where another tablet started the workflow and a reparent occurred,
+// and now that we manage the workflow, we need to read the rows_copied that already exists and add them to our
+// counter, otherwise it will look like the reparent wiped all the rows_copied.
+// So in the event that our CopyRowCount counter is zero, and the existing rows_copied in the vreplication table
+// is not, copy the value of vreplication.rows_copied into our CopyRowCount
```

Comment:
Non blocking nit, but IMO we should make the lines shorter and we should end both sentences with punctuation (we don't for the last one).

```suggestion
// setExistingRowsCopied deals with the case where another tablet started
// the workflow and a reparent occurred, and now that we manage the
// workflow, we need to read the rows_copied that already exists and add
// them to our counter, otherwise it will look like the reparent wiped all the
// rows_copied. So in the event that our CopyRowCount counter is zero, and
// the existing rows_copied in the vreplication table is not, copy the value of
// vreplication.rows_copied into our CopyRowCount.
```